lib/season_stats_module.rb

```diff
@@ -39,33 +39,43 @@ def worst_coach(season_id)
     end[0]
   end
 
-  def most_accurate_team(season_id)
-    team_hash = Hash.new
-    season_teams(season_id).each do |team|
-      total_shots = game_teams_data_for_season(season_id).sum do |game|
-        if game.team_id == team
-          game.shots
-        else
-          0
-        end
+  def total_shots_by_team(season_id, team)
+    total_shots = game_teams_data_for_season(season_id).sum do |game|
+      if game.team_id == team
+        game.shots
+      else
+        0
       end
+    end
+    total_shots
+  end
 
-      total_goals = game_teams_data_for_season(season_id).sum do |game|
-        if game.team_id == team
-          game.goals
-        else
-          0
-        end
+  def total_goals_by_team(season_id, team)
+    total_goals = game_teams_data_for_season(season_id).sum do |game|
```

Comment:
I think you could take out the local variable and just return the value of the sum.

Reply:
Nice eye, it has been removed for this method and the total shots method.